syncstorage/src/tokenserver/settings.rs

```diff
@@ -102,6 +104,7 @@ impl Default for Settings {
             run_migrations: cfg!(test),
             spanner_node_id: None,
             additional_blocking_threads_for_fxa_requests: None,
+            token_duration: 3600,
```

Comment:
In some other repos, we put constants like this into their own file or block, so that they're easier to find. 
Might be a follow up issue and a #goodfirstbug .